source/server/server.cc

```diff
@@ -363,6 +363,8 @@ void InstanceImpl::initialize(const Options& options,
       component_factory.createRuntime(*this, initial_config));
   hooks.onRuntimeCreated();
 
+  // Initialize the dynamic runtime now that we have the static base runtime.
```

Comment:
?

Reply:
This is some part of TDS that escaped my patch splitting, will remove.